libs/remixd/src/websocket.ts

```diff
@@ -1,24 +1,24 @@
 import * as WS from 'ws'
 import * as http from 'http'
-import { WebsocketOpt, SharedFolderClient } from './types' // eslint-disable-line
+import { WebsocketOpt, ServiceClient } from './types' // eslint-disable-line
```

Comment:
any reason why you need to disable line ? I'm not sure you need to do that anymore.